### PR TITLE
Change metric type from Gauge to Counter

### PR DESCRIPTION
**Pull request by chaosaffe**
Currently the metrics leveraged are of a gauge type, even though they are presenting counter data (monotonically increasing values) which can then have the family of `rate()` functions applied to extract instantaneous and trend data.

### CODE REVIEW

File: gorush/metrics.go [hunk 1]
```diff
@@ -81,42 +81,42 @@ func (c Metrics) Describe(ch chan<- *prometheus.Desc) {
...
 	)
 	ch <- prometheus.MustNewConstMetric(
 		c.HuaweiError,
-		prometheus.GaugeValue,
+		prometheus.CounterValue,
 		float64(StatStorage.GetHuaweiError()),
 	)
 	ch <- prometheus.MustNewConstMetric(
 		c.QueueUsage,
-		prometheus.GaugeValue,
```
Comment (appleboy) [inline gorush/metrics.go:119]: I think `queue usage` should not change to counter value type.

Comment (chaosaffe) [reply gorush/metrics.go:119]: Good point, that one should remain a gauge

Comment (appleboy) [reply gorush/metrics.go:119]: @chaosaffe Please revert this line. Thanks

**Comment by appleboy** [comment]
https://cloud.drone.io/appleboy/gorush/539/1/3 build error.

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
@chaosaffe THanks.

**Pull request merged**